helpers/markets/bsctestnet.ts

```diff
@@ -634,4 +634,26 @@ export default [
       vTokenReceiver: "VTreasury",
     },
   },
+  {
+    name: "Venus PT-USDe-30Oct2025",
```

Comment:
We should use `OCT` instead of `Oct` for mainnet. Not needed to change it on testnet